src/main/java/Main.java

```diff
@@ -120,8 +120,8 @@ public static void main(String[] args) throws Exception {
             String v = System.getProperty("java.class.version");
             if (v!=null) {
                 try {
-                    if (Float.parseFloat(v)<52.0f)
-                        throw new UnsupportedClassVersionError(v);
+                    if (Float.parseFloat(v)!=52.0f)
+                        throw new UnsupportedClassVersionError(v);   
```

Comment:
OMG, not your change I know, but well, we really throw this ourselves?... All my basics are being challenged :)